README.md

```diff
@@ -1,3 +1,55 @@
-# README
+# Respond to an Employment Tribunal Claim
+[![Build Status](https://travis-ci.org/ministryofjustice/et3.svg?branch=develop)](https://travis-ci.org/ministryofjustice/et3)
 
-This app is currently in development and not ready for use or deployment. It will replace the current online version of the ET3 form.
\ No newline at end of file
+Digital ET3 form within the in-house ET service, replacing a third party supplier.
+
+## Getting Started
+This is a Ruby on Rails app which uses [GOV.UK Elements](https://github.com/alphagov/govuk_elements) and sits within the ET service.
+The form submits to the [ET API](https://github.com/ministryofjustice/et_api) which can only be done via the [ET Full System](https://github.com/ministryofjustice/et-full-system).
+Running this app will enable development of the app itself, with its test suite mocking the final submission. 
+
+### Prerequisites
+* Ruby 2.5.1 (we recommend using [RVM](https://rvm.io/))
+* [Rails 5.2](https://rubygems.org/gems/rails)
```

Comment:
Rails isn't a prerequisite - its built in ?

Reply:
Yeah I know it gets installed with `bundle install` but I wasn't sure if I should leave it out of the documentation? I'm tempted to leave it in just for clarity, even though it's not necessary.